docs/source/index.rst

```diff
@@ -0,0 +1,53 @@
+.. Django Multi-tenant documentation master file, created by
+   sphinx-quickstart on Mon Feb 13 13:32:28 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Django Multi-tenant's documentation!
+=================================================
+
+|Latest Documentation Status| |Build Status| |Coverage Status| |PyPI Version|
```

Comment:
This is pretty cool, what about putting these badges in the README as well?

Reply:
Added